convert2rhel/checks.py

```diff
@@ -295,6 +296,17 @@ def ensure_compatibility_of_kmods():
         logger.info("Kernel modules are compatible.")
 
 
+def validate_yum_transaction():
+    """ """
+    logger.info("\nValidate the %s transaction", pkgmanager.TYPE)
+
+    pkghandler.transaction_handler.process_transaction(
+        test_transaction=True,
+    )
+
+    logger.info("Transaction validated successfully.\n")
```

Comment:
I would like some suggestions on how to improve this. Currently, I'm not sure how to handle the case where we want to test the transaction first, and then process it later, since it's the same code. 

Should I introduce a new function just to test and `try-except` the exceptions that arise there? Probably a discussion about this point would be valuable from the POV of making it more reliable in both "phases".

Reply:
Was this discussed in our cabal? from my POV unless we know of an exception that can raise which we don't want to care about when validating, I wouldn't want a try-catch created

Reply:
Unfortunately, not yet. But I can bring this to the next cabal... I will add a comment to the document later.

Reply:
Discussed on Cabal: Oct 4, 2022